contracts/ROSCA.sol

```diff
@@ -299,7 +299,7 @@ contract ROSCA {
       address candidate = membersAddresses[index];
       if (!members[candidate].paid) {
         winnerIndex = index;
-        if (members[candidate].credit + totalDiscounts >= requiredContribution()) {
+        if (userTotalCredit(candidate) >= requiredContribution()) {
```

Comment:
I thought we'd like to check something like: 

```
members[memberAddress].credit >= requiredContribution()
```
the amount user has contributed so far against what's the amount he/she's supposed to contributed up to current round? 

Why the user credit has to have surplus (which is: totalDiscounts / membersAddresses.length) ?

Reply:
traditional ROSCA have this logic where if you are supposed to pay 1 ETH per round, and you have 0.1 ETH discount (earned from the last round). you only need to pay up 0.9 ETH this round.